### PR TITLE
T10 f22 20 profile add profile bio to profile page

### PR DESCRIPTION
**Pull request by Zevez7**
completed add profile bio to profile page. made some changes to the profile model to add firstname, lastname and biography

### CODE REVIEW

**Review by thematthewchan — APPROVED** [review]
Changes look good to me.

**Comment by Zevez7** [comment]
3 days already commiting without second review to keep project going

**Pull request merged**
